screwdriver.yaml

```diff
@@ -11,6 +11,11 @@ jobs:
         steps:
             - vet: go vet ./...
             - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
+            - pretest-setup: |
+                  wget -O hab.tar.gz 'https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-0.79.1-20190410220617-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux'
+                  tar -C . -ozxvf hab.tar.gz
+                  mv hab-*/hab /hab/bin/hab
+                  /hab/bin/hab pkg install -bf core/zstd
```

Comment:
won't we have this in launcher already ?

Reply:
this is for pr tests in store-cli